tests/pytests/test.py

```diff
@@ -3807,40 +3868,7 @@ def prepare_env(env):
                    'UNIXADDR',
                    '/tmp/redis.sock',
                    'MASTER'
-            ).error().contains('Invalid tcp address').contains(addr)
-
-def test_cluster_set_server_memory_tracking(env):
-    if not env.isCluster():
-        # this test is only relevant on cluster
-        env.skip()
-
-    def get_memory(env):
-        res = env.cmd('INFO', "MEMORY")
-        return res['used_memory']
-
-    def cluster_set_ipv4():
-        env.cmd('SEARCH.CLUSTERSET',
-               'MYID',
-               '1',
-               'RANGES',
-               '1',
-               'SHARD',
-               '1',
-               'SLOTRANGE',
-               '0',
-               '16383',
-               'ADDR',
-               'password@127.0.0.1:22000',
-               'MASTER'
-            )
-    for _ in range(200):
-        cluster_set_ipv4()
-    initial = get_memory(env) - 1024 * 1024 # to account for some variance in memory
-    for _ in range(1_000): # hangs at 1932 iterations. need to determine the cause
-        cluster_set_ipv4()
-        mem = get_memory(env)
-        env.assertLessEqual(initial, mem)
-
+            ).error().contains('Bad value for ADDR:').contains(addr)
```

Comment:
Should we add test that validate that valid address are parsed as expected (including password)? Maybe we already have such test.